pilot/pkg/bootstrap/server.go

```diff
@@ -1061,30 +1066,42 @@ func (s *Server) maybeCreateCA(caOpts *CAOptions) error {
 		if s.kubeClient != nil {
 			corev1 = s.kubeClient.CoreV1()
 		}
-		// May return nil, if the CA is missing required configs - This is not an error.
+		// TODO: Issue #27606 If External CA is configured, use that to sign DNS Certs as well. IstioCA need not be initialized
 		if s.CA, err = s.createIstioCA(corev1, caOpts); err != nil {
 			return fmt.Errorf("failed to create CA: %v", err)
 		}
+		if caOpts.CAType != "" {
```

Comment:
Can the istio CA and RA initiated simultaneously?

Reply:
Not exactly. Currently, this PR only ensures that workload certificates are signed by the external CA, not the DNS certificates issued to Istiod - that is controlled by a different config. the createIstioCA code creates the self signed certificates and others required to sign DNS certificates. This will be addressed in a future PR